asmcomp/power/emit.mlp

```diff
@@ -472,19 +394,20 @@ module BR = Branch_relaxation.Make (struct
                hi >= -0x8000 && hi <= 0x7FFF) then 2
       else if (let (_lo, hi) = native_low_high_u n in
                hi >= -0x8000 && hi <= 0x7FFF) then 2
```

Comment:
Can these high/low checks also be removed since tocload_size is always 2 now?

Reply:
That suddenly made sense when I split my editor and put both functions beside each other! I withdraw my comment.